ssm-qan-api.spec missing

I've taken a fresh clone of the ssm-submodules repository then attempted to build the ssm-qan-api srpm, and I get the following error:

```
$ make rpms packages=ssm-qan-api
...
+ git checkout v1.17.4
Updating files: 100% (4160/4160), done.
Previous HEAD position was 267835e Merge pull request #4 from shatteredsilicon/update-version
HEAD is now at 44dc676 Update README.md
+ sed 's/%{_version}/v1.17.4/g' ssm-qan-api.spec
sed: can't read ssm-qan-api.spec: No such file or directory
make: *** [Makefile:9: srpms] Error 2
```
I've attached the full build output: [ssm-qan-api.log](https://github.com/shatteredsilicon/ssm-submodules/files/9394477/ssm-qan-api.log)

--- a/ssm-qan-api.spec
+++ b/ssm-qan-api.spec
@@ -43,11 +43,11 @@ cp -r ${HOME}/go/src/%{provider_prefix}/vendor/github.com/revel ${HOME}/go/src/g
 
 %build
 mkdir -p bin release
+cp -r $(go env GOROOT) ${HOME}/goroot
+export GOROOT=${HOME}/goroot
 export GOPATH=${HOME}/go
 export APP_VERSION="%{version}"
 GO111MODULE=off go build -o ./revel ${GOPATH}/src/%{provider_prefix}/vendor/github.com/revel/cmd/revel
-#find ${GOPATH}/src/%{provider_prefix}/vendor -name "testdata" -type d -prune -exec rm -rf "{}" \;
-#rm -rf ${GOPATH}/src/%{provider_prefix}/vendor/golang.org/x/tools/gopls
 GO111MODULE=off ./revel build %{provider_prefix} release prod
 rm -rf release/src/github.com/shatteredsilicon/qan-api
 mkdir -p ./src

